src/loaders/svg/tvgXmlParser.cpp

```diff
@@ -176,20 +240,28 @@ bool simpleXmlParseAttributes(const char* buf, unsigned bufLength, simpleXMLAttr
             valueEnd = _simpleXmlFindWhiteSpace(value, itrEnd);
         }
 
+        itr = valueEnd + 1;
+
+        value = _skipWhiteSpacesAndXmlEntities(value, itrEnd);
+        valueEnd = _unskipWhiteSpacesAndXmlEntities(valueEnd, value);
+
         memcpy(tmpBuf, key, keyEnd - key);
         tmpBuf[keyEnd - key] = '\0';
 
         tval = tmpBuf + (keyEnd - key) + 1;
-        memcpy(tval, value, valueEnd - value);
-        tval[valueEnd - value] = '\0';
+        int i = 0;
+        while (value < valueEnd) {
```

Comment:
Why need this logic? 
Entities are already skipped from 245 line.

Reply:
we need this to remove all entities from the middle part of the value, like in the example that we had:
mask="url(&quot;#mask&quot;)"